packages/server-ct/src/project-ct.ts

```diff
@@ -89,6 +89,13 @@ export class ProjectCt extends ProjectBase<ServerCt> {
 
       updatedConfig.componentTesting = true
```

Comment:
So this was the problem!

Interesting. I think it's still a good practice to encourage people to return the config regardless, but good job tracking this one down.

Reply:
Why is that a good practice? I don't understand.